src/unum/wireless/wireless_nl80211_lede.c

```diff
@@ -0,0 +1,25 @@
+#include "unum.h"
+// All functions dependent on LEDE specific platform
+// Given phy name, get the first interface (VAP) name
+// Since the phy to interface mapping is maintained in wireless_iwinfo,
+// we use iwinfo APIs
```

Comment:
Please make a note that the agent nl80211 interfaces should eventually be completely decoupled from iwinfo based ones.

Reply:
Added a note.